gtsam/3rdparty/metis/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.8)
```

Comment:
We can't make changes inside metis. Licensing and upgrade difficulties.

Reply:
Interesting. I saw the git-blame for this CMake file and this file has been updated before, hence I went ahead. Does this new information change anything?

Reply:
Actually, maybe the Makefile is what we added, and we should not make changes in the library itself.

Reply:
Yup, the library is unchanged. :slightly_smiling_face: Guess we'll keep this update.